slides.md

```diff
@@ -275,41 +275,58 @@ const todos = [{
 Pick one of the two options and get rid of the warning, then remove the property `key` from the `TodoItem` element.
 
 ---
-# State & co
+# State
 It's time to tick some TODOs, let's add a new field to them: `completed`, and set it to `false`.
 
-Now, let's add that to our properties in `Todo`, and let's use a `Switch` to display it. The property to tell a `Switch` if it's active or not is `value`.
+```javascript
+const todos = [{
+  name: "Take the dog out",
+  completed: false
+}];
+```
 
----
-# Tick a TODO! {.big}
+Now, let's add that to our properties in `Todo`:
 
----
-# It lives..!
-![](https://media.giphy.com/media/2wwJpbHZeSQ5G/giphy.gif)
+```javascript
+const Todo = ({name, completed}) => ...
+```
 
----
-# ... Or not?
-![](https://media.giphy.com/media/MpceqYw3yRC9y/giphy.gif)
+To display the TODO being completed or not, we can use a `Switch`, which accepts a property named `value`.
```

Comment:
Maybe "to toggle between the TODO..."

Reply:
Hm, I see what you mean, but I want to focus on the display part (you can't mark the todo as completed, yet), maybe "To display the two possible states of the TODO"?

Reply:
36de041